dist/dbus-1/system.d/org.coreos.zincati1.conf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+          "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+          "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+
+  <!-- Only user zincati can own the service -->
+  <policy user="zincati">
+    <allow own="org.coreos.zincati1"/>
```

Comment:
This is cosmetic of course, but IMO the chance of us ever introducting a `zincati2` bus name is zero.  Personally I'd just name this `zincati` - we don't expect to make this a widely used stable API anyways.

From a quick inspection here in `busctl`: systemd, rpm-ostree and firewalld use the `1` suffix, other services like NetworkManager, fwupd, don't.  But tellingly there are zero `2` or more services :smile: